modules/ROOT/pages/xamarin/using-metrics-sdk.adoc

```diff
@@ -0,0 +1,24 @@
+= AeroGear Services Metrics SDK
+
+This document explains how to use the AeroGear Metrics SDK on the Android platform and integrate it with an App.
```

Comment:
This file is located within the context of 'xamarin' yet 'Android' is mentioned in the title. Should the state: 'xamarin' ?